testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableCompatibility.java

```diff
@@ -312,6 +313,26 @@ public void testCreateTableWithColumnCommentOnDelta()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
```

Comment:
We should be able to run this test on OSS Delta Lake.

Reply:
Yes, I've moved this to a separate test class where it is enabled for OSS Delta Lake. Is there a simple way to run the product tests locally?